internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -405,6 +416,7 @@ func (d *dataPlaneKafkaService) unassignKafkaFromDataplaneCluster(kafka *dbapi.K
 		}
 
 		metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusProvisioning, kafka.ID, kafka.ClusterID, time.Since(kafka.CreatedAt))
+		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusProvisioning, kafka.ID, kafka.ClusterID)
```

Comment:
Same comment as above. We'll lose track of the time this Kafka instance spent in this state before this data plane cluster unassignment.